### DISCUSSION TITLE
Docker Step by Step

### DISCUSSION THREAD

**Discussion by DanOHH** [wotschofsky/discord-live-translator#97] [Q&A] [answered]
Hi Felix,

extremly excited i found your solution for our discord server, where we have multi languages peoples and sometimes the english skills aren't the best / they are at their limits. Therfore, I installed / added the free version to our server, but got the "no service / doesnt react message". So I checked out your GitHub and I saw the "docker" solution, but I'm totally lost what i have to do.

I Installed the Docker software and tried to import the "dockerfile" but I'm not abel to do so. :(

Would it be possibile, that you can provide me and the rest of the world a short "How to for dummies?"

Thank you very much and have a great week :D
DanO

**Comment by wotschofsky** [comment, marked as answer]
Hi, basically you'll need to only run the following commands in your terminal, after installing Docker on your machine. I'll assume that you know how to do this.

1. `git clone https://github.com/wotschofsky/discord-live-translator`
2. `cd discord-live-translator`
3. Then create a file called `.env` in that same directory. The contents should be `BOT_TOKEN=YOURDISCORDTOKENHERE`. Remember to put in your token.
4. `docker compose -f docker-compose.prod.yml up -d`

Then you're done. Hope this helps!
